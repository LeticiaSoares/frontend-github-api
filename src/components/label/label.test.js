import React from 'react'
import { shallow } from 'enzyme';
import Label from './index'
import 'jest-styled-components'

describe('<Label />', () => {
    it('should match default snapshot', () => {
        const component = shallow(<Label />)
        expect(component).toMatchSnapshot()
    })
    it('should have the right displayName', () => {
        expect(Label.displayName).toEqual('Label')
    })
})
