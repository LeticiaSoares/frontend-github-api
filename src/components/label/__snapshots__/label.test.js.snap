// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> should match default snapshot 1`] = `ShallowWrapper {}`;
